lib/fathead/htmlref/cover/tags.txt

```diff
@@ -0,0 +1,142 @@
+a
+abbr
+address
+area
+article
+aside
+audio
+b
+base
+bdi
+bdo
+blockquote
+body
+br
+button
+button type=submit
```

Comment:
@sriranganathan do we have an article that matches this? or do we just have an article for `button`. These should all match an article titles we expect to see in the `output.txt` -- think of these as unit tests. So in the future if we run the parser again and for some reason we don't have an article for `button`, we'll know something is wrong 👍